spec/wfx.openapi.yml

```diff
@@ -91,11 +91,6 @@ paths:
                     type: string
                     example: 25f4dd3f9828553c9e437a24f7fa451ae60916f2
                     x-go-type-skip-optional-pointer: true
-                  buildDate:
```

Comment:
Removing this is a "(half) breaking" change, not in so far as endpoints change – which would be a "really" breaking change – but still, contents change. As the build date should not have been relied upon for anything anyway and has just informational value, I guess it's fine to just remove it with an according note in the release change logs.